### PR TITLE
Added scraped date

### PR DESCRIPTION
**Pull request by danielacraciun**

### CODE REVIEW

File: scrape_rec/db_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    __tablename__ = POSTGRES_DB
+
+    fingerprint = sa.Column(sa.String, primary_key=True)
+
+    # Mandatory
+    title = sa.Column(sa.String, nullable=False)
+    description = sa.Column(sa.String, nullable=False)
+    scraped_date = sa.Column(sa.DateTime, nullable=False)
+    price = sa.Column(sa.Integer, nullable=False)
+    currency = sa.Column(sa.String, nullable=False)
```
Comment (mirceachira) [inline scrape_rec/db_wrapper.py]: Does this default to euro? It feels like it should default to euro as it's usually the case

File: scrape_rec/utils.py [hunk 2]
```diff
@@ -1,74 +1,22 @@
...
-def get_postgres_session():
-    # To fix db table changes errors enable once
-    # db.execute('DROP TABLE realestate;')
-    base.metadata.create_all(db)
-    Session = sa.orm.sessionmaker(db)  
-    return Session()
+import pickle
+from scrapy.http import Headers
+from scrapy.responsetypes import responsetypes
+from w3lib.http import headers_raw_to_dict
```
Comment (mirceachira) [inline scrape_rec/utils.py]: are these used?

File: scrape_rec/db_wrapper.py (hunk 1 above)
Comment (danielacraciun) [reply scrape_rec/db_wrapper.py]: I wouldn't default, I'd rather drop the item.

File: scrape_rec/utils.py (hunk 2 above)
Comment (danielacraciun) [reply scrape_rec/utils.py]: Think this were used for the cache retrievals? Maybe we should leave them here :wink:

**Pull request merged**